app/logical/source/extractor/bluesky.rb

```diff
@@ -117,6 +113,14 @@ def html_artist_commentary_desc
       text[byte_start...byte_end] = %{<a href="https://bsky.app/hashtag/#{CGI.escapeHTML(Danbooru::URL.escape(tag_name))}">##{CGI.escapeHTML(tag_name)}</a>}.force_encoding("ASCII-8BIT")
     end
 
+    api_response&.dig("thread", "post", "record", "embed", "images").to_a.pluck(:alt).compact_blank.each do |alt_text|
```

Comment:
Embed is potentially a `app.bsky.embed.recordWithMedia` (see above). This also assumes the embed is images and not video/something else

Reply:
I see what you mean, I'll fix that. Do you have an example for videos with an alt text, for testing?

Reply:
I don't know where it shows it in the UI but this video has alt text:
https://bsky.app/profile/rningscissors.bsky.social/post/3lozjurmajk25

Reply:
Apparently their web player does not have support for it (https://github.com/bluesky-social/social-app /issues/5959). It's pretty funny that with this pull we'll have more support for their own platform features than they do.